frontend/src/patientApp/timeOfTest/DOB.tsx

```diff
@@ -15,8 +15,18 @@ import { dateFromStrings } from "../../app/utils/date";
 const DOB = () => {
   const { t } = useTranslation();
 
-  const dispatch = useDispatch();
   const plid = useSelector((state: any) => state.plid);
+
+  useEffect(() => {
+    PxpApi.getObfuscatedPatientName(plid)
```

Comment:
Is this the only place we expect to use this `getObfuscatedName` API call? If so, I'm wondering if it makes more sense to update the `getPatientLinkVerify` API endpoint to return both the existing PxpResponse object and the patient's obfuscated name. It looks to me like we'll be calling this name endpoint on every pxp view, so it might be worth keeping it all to a single logical endpoint.

Reply:
If I'm understanding you correctly: the `GET /pxp/patient-name` call happens _before_ the user attempts DOB verification, in order to provide a hint to the patient that they're looking at the right test result. The `getPatientLinkVerify` call is only performed later, upon form submission.

Reply:
Ahh yes, of course. In that case this looks good 👍